setup.py

```diff
@@ -87,7 +86,7 @@ def run_tests(self):
 
 setup(
     name='django-richenum',
-    version='3.8.0',
+    version='3.9.0',
```

Comment:
Good callout. I was overruling myself because django has introduced more than a few breaking changes in minor versions, but that's no excuse to do the same here 👍🏻